db/seeds.rb

```diff
@@ -1,7 +1,19 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+connection = ActiveRecord::Base.connection
+
+%w[states districts zip_codes districts_zip_codes legislators].each do |table|
+  sql = File.read("db/seed_data/#{table}.sql")
+  statements = sql.split(/;$/)
+
+  ActiveRecord::Base.transaction do
```

Comment:
Did you try `ActiveRecord::Base.connection.execute(IO.read("db/seed_data/#{table}.sql"))`?  That would avoid ruby having to parse the individual lines and probably speed it up a little.  

This merge request looks good.  I don't love manually assigning the sequences, but unless we wanted to make a lot of changes and create all the records w/ rails, I cannot think of a better way to do it. 

I'm going to go ahead and merge.  Maybe try that out the one liner above if you have time late.